src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories.html.twig

```diff
@@ -32,34 +32,29 @@
       <i class="material-icons arrow-down float-right">keyboard_arrow_down</i>
     </div>
 
-    <div class="ps-dropdown-menu dropdown-menu module-category-selector" aria-labelledby="{{refMenu}}">
-      <ul class="items-list js-items-list">
-        <li class="module-category-reset">
-          <a class="dropdown-item" href="#">
-            {{ 'All Categories'|trans({}, 'Admin.Modules.Feature') }}
-          </a>
-        </li>
-        <li class="module-category-menu module-category-recently-used"
-          data-category-ref="recently-used"
-          data-category-display-name="{{ 'Recently Used'|trans({}, 'Admin.Modules.Feature') }}"
-        >
-          <a class="dropdown-item" href="#">
-            {{ 'Recently Used'|trans({}, 'Admin.Modules.Feature') }}
-          </a>
-        </li>
-        </li>
-        {% for subMenu in menu.subMenu %}
-          <li class="module-category-menu"
+    <div class="ps-dropdown-menu dropdown-menu module-category-selector items-list js-items-list" aria-labelledby="{{refMenu}}">
+      <a class="dropdown-item module-category-reset">
+        {{ 'All Categories'|trans({}, 'Admin.Modules.Feature') }}
+      </a>
+
+      <a 
+        class="dropdown-item module-category-menu module-category-recently-used"
+        data-category-ref="recently-used"
+        data-category-display-name="{{ 'Recently Used'|trans({}, 'Admin.Modules.Feature') }}"
+      >
+        {{ 'Recently Used'|trans({}, 'Admin.Modules.Feature') }}
+      </a>
+
+      {% for subMenu in menu.subMenu %}
+          <a 
+            class="dropdown-item module-category-menu"
             {% if subMenu.tab %}data-category-tab="{{subMenu.tab}}"{% endif %}
             data-category-ref="{{subMenu.refMenu}}"
             data-category-display-name="{{subMenu.name|trans({}, 'Admin.Modules.Feature')}}"
           >
-            <a class="dropdown-item" href="#">
-              {{subMenu.name|trans({}, 'Admin.Modules.Feature')}}<span class="float-right">{{subMenu.modules|length }}</span>
-            </a>
-          </li>
-        {% endfor %}
-      </ul>
+            {{subMenu.name|trans({}, 'Admin.Modules.Feature')}}<span class="float-right">{{subMenu.modules|length }}</span>
```

Comment:
I can see "trans(" and a domain (Admin.Modules.Feature) but I cannot see any wording. Is it normal?

Reply:
The wording is coming from the backend, wdyt @PrestaShop/prestashop-maintainers ?

Reply:
That’s because the string being translated is dynamically passed to the translator. So you can only say if the domain is right

Reply:
Mhm, I'm not sure to understand how I can say if the domain is right without the wording? 🤔 I can read "subMenu.name" so if it is a "title" then yes it should be in Admin.Modules.Feature